### DISCUSSION TITLE
Perdita associazione fattura-allegato con utilizzo API received_documents (PUT)

### DISCUSSION THREAD

**Discussion by ladome** [fattureincloud/api#302] [Q&A] [answered]
Stiamo riscontrando dei problemi con l'integrazione API.

Modify Received Document

**PUT** _/c/{company_id}/received_documents/{document_id}_

Ad esempio nella chiamata passiamo questa richiesta:

```
{
  "data": {
    "id": "208663302",
    "description": "descrizione",
    "rc_center": "2023"
}
```

Come abbiamo sempre fatto, passiamo solo questi 3 campi che sono soggetti a nostra modifica, in modo tale da lasciare inalterati tutti gli altri.
Ad oggi riscontriamo che a seguito di questa chiamata l'associazione con la fattura allegata viene persa. (allegato che può essere o un documento caricato o una una fattura elettronica)

Di seguito gli step per replicare la problematica:
- Acquisto creato in piattaforma
- Modifica fatta in piattaforma
- Modifica tramite API passando i 3 campi descritti sopra
- Dopo modifica API viene persa l'associazione con la fattura allegata.

Potete fare una verifica e darci un rapido riscontro visto che la problematica ci sta creando diversi disagi?

**Comment by emajo** [comment]
Ciao @ladome, ci può fornire un esempio di creazione così possiamo fare qualche test per replicare il problema?

Grazie,
Emanuele.

**Reply by hierro1981** [reply]
Ciao @emajo, puoi creare una fattura qualsiasi (sia usando un file caricato che fattura elettronica). Il problema è indipendente da come viene creata la fattura.

**Comment by Enricond** [comment]
confermo che succede anche a me. 
se modifico la descrizione del documento con lo stesso metodo di ladome l'allegato viene rimosso

**Comment by emajo** [comment]
Grazie a tutti per la segnalazione, stiamo lavorando per risolvere il problema.

**Reply by hierro1981** [reply]
Potete condividere delle tempistiche di risoluzione?

**Comment by emajo** [comment, marked as answer]
Abbiamo sistemato il problema, tutto dovrebbe funzionare correttamente ora.

Grazie ancora a tutti,
Emanuele.
